packages/react-components/react-badge/src/components/Badge/useBadgeStyles.ts

```diff
@@ -12,16 +12,35 @@ export const badgeClassNames: SlotClassNames<BadgeSlots> = {
 // Instead, add extra padding to the root, and a negative margin on the icon to "remove" the extra padding on the icon.
 const textPadding = tokens.spacingHorizontalXXS;
 
-const useRootStyles = makeStyles({
-  base: {
-    display: 'inline-flex',
-    boxSizing: 'border-box',
-    alignItems: 'center',
-    justifyContent: 'center',
-    position: 'relative',
-    ...typographyStyles.caption1Strong,
+const useRootClassName = makeResetStyles({
+  display: 'inline-flex',
+  boxSizing: 'border-box',
+  alignItems: 'center',
+  justifyContent: 'center',
+  position: 'relative',
+  ...typographyStyles.caption1Strong,
+  height: '20px',
+  minWidth: '20px',
+  padding: `0 calc(${tokens.spacingHorizontalXS} + ${textPadding})`,
+  borderRadius: tokens.borderRadiusCircular,
+  // Use a transparent stroke (rather than no border) so the border is visible in high contrast
+  borderColor: tokens.colorTransparentStroke,
+
+  '::after': {
+    content: '""',
+    position: 'absolute',
+    top: 0,
+    left: 0,
+    bottom: 0,
+    right: 0,
+    borderStyle: 'solid',
+    borderWidth: tokens.strokeWidthThin,
+    borderColor: 'inherit',
```

Comment:
_nit:_ You can reduce this to one line:

```suggestion
    border: `${tokens.strokeWidthThin} solid inherit`,
```

Reply:
I made that change and got a bunch of VR test failures. Turns out `inherit` cannot be used in CSS shorthands.

[From MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/Shorthand_properties):

> Only the individual properties values can inherit. As missing values are replaced by their initial value, it is impossible to allow inheritance of individual properties by omitting them. The keyword inherit can be applied to a property, but only as a whole, not as a keyword for one value or another. That means that the only way to make some specific value to be inherited is to use the longhand property with the keyword inherit.

[Codepen example](https://codepen.io/seanms/pen/wvEwOMY).